dist/index.html

```diff
@@ -11,17 +11,18 @@ <h1 id="leader">Leaderboard</h1>
     <div class="scores-section">
     <div class="heading">
       <h2>Recent scores</h2>
-      <div id="btn-refresh"><button type="submit" class="refresh">Refresh</button></div>
+      <div id="btn-refresh" class="refresh"><button type="submit" class="refresh">Refresh</button></div>
     </div>
     <div id="list" class="list"></div>
     </div>
       <div class="leader-board" id="leader-board">
         <h2>Add your score</h2>
-        <form name="board" id="form">
-          <input type="text" id="name" name="name" placeholder="Your name"><br><br>
-          <input type="text" id="score" name="score" placeholder="Your score"><br><br>
+        <form name="board" id="form" class="form" action="submit">
+          <input type="text" id="name" name="name" class="forminput" placeholder="Your name"><br><br>
+          <input type="number" id="score" name="score" class="forminput" placeholder="Your score"><br><br>
```

Comment:
OPTIONAL

- [ ] Kindly add validations to your form input `required` so a user can't submit empty values 👍🏿